react-ui/src/components/splash-banner.jsx

```diff
@@ -1,21 +1,21 @@
-import React from 'react';
-
-import logo from '../assets/logo-colored-horizontal.png';
-import '../styles/App.css';
-
-
-
-function HomeUI() {
-  return (
-    <div className="HomeUI">
-      <header className="splash-header">
-        <img src={logo} className="logo" alt="Nainers Matter logo" />
-        <p>
-          Full site coming soon!
-        </p>
-      </header>
-    </div>
-  );
-}
-
-export default HomeUI;
\ No newline at end of file
+import React from 'react';
+
+import logo from '../assets/logo-colored-horizontal.png';
+
+
+
+
+function HomeUIWrapper() {
+  return (
+    <div className="home-ui-wrapper">
+      <header className="splash-header">
+        <img src={logo} className="logo" alt="Nainers Matter logo" />

```

Comment:
Nainers? 😝

Reply:
the component isn't being used at the moment - so meh.

Good eye though!!